src/components/orgs/Coupon/List.tsx

```diff
@@ -57,6 +57,16 @@ const CouponList: React.FC<ICouponListProps> = ({ supplierId }) => {
     (coupon) => coupon.status === "ACTIVE",
   );
 
+  function filterLimitReachedCoupons(coupons: ICoupon[]) {

```

Comment:
Please follow newest pattern for functions:

```suggestion
  const filterLimitReachedCoupons = (coupons: ICoupon[]) => {
```

Also suggestion for name: 
```suggestion
  const filteredCoupons = (coupons: ICoupon[]) => {
```

Which can be used in the future for other filter rather then only maxTotal